output/roundrobin.md

```diff
@@ -11,20 +11,32 @@ It is included in Fluentd's core.
   @type roundrobin
 
   <store>
-    @type tcp
-    host 192.168.1.21
+    @type http
+    endpoint http://192.168.1.21
+    <format>
```

Comment:
`format` and ` json_array` aren't needed, because they aren't mandatory options.
Since this page is a document for roundrobin plugin, information that is not related to it should be suppressed as much as possible.

Reply:
It should be, thanks!